remoteappmanager/docker/container_manager.py

```diff
@@ -376,8 +386,9 @@ def _get_ip_and_port(self, container_id):
         # docker url to extract the ip.
         ip = '127.0.0.1'
 
-        if self.docker_config and self.docker_config.docker_host != '':
-            url = urlparse(self.docker_config.docker_host)
+        base_url = self.docker_config.get("base_url")
+        if base_url:
+            url = urlparse(base_url)
             if url.scheme == 'tcp':
```

Comment:
https://github.com/simphony/simphony-remote/pull/126/files#r71502734 refers to here.  Maybe we should take the hostname regardless of its scheme?